# Author: Jiri Furst <Jiri.Furst@gmail.com>
# Allow oversubscribe for tests on systems with less than 8 CPUs
--- dakota-6.19.0-public-src-cli/test/dakota_test.perl.orig     2024-01-17 17:27:03.932756724 +0100
+++ dakota-6.19.0-public-src-cli/test/dakota_test.perl  2024-01-17 17:27:46.878513897 +0100
@@ -1088,7 +1088,7 @@
     { 
       # default for Linux
       $test_command = 
-        "mpirun -np $num_proc $fulldakota $redir";
+        "mpirun --oversubscribe -np $num_proc $fulldakota $redir";
     }
   }
 

